go.mod

```diff
@@ -1,60 +1,63 @@
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/go-chi/chi v4.0.4+incompatible
+	github.com/go-openapi/errors v0.19.4 // indirect
+	github.com/go-openapi/runtime v0.19.14
+	github.com/go-openapi/strfmt v0.19.5 // indirect
+	github.com/go-openapi/swag v0.19.8 // indirect
 	github.com/gobuffalo/packd v1.0.0
-	github.com/gobuffalo/packr/v2 v2.7.1
-	github.com/hashicorp/go-immutable-radix v1.0.0
-	github.com/hashicorp/golang-lru v0.5.3
+	github.com/gobuffalo/packr/v2 v2.8.0
+	github.com/hashicorp/go-immutable-radix v1.2.0
+	github.com/hashicorp/golang-lru v0.5.4
 	github.com/influxdata/influxdb v1.7.9
 	github.com/jacobsa/daemonize v0.0.0-20160101105449-e460293e890f
-	github.com/jacobsa/fuse v0.0.0-20200311085126-7d791d27a25d
-	github.com/jstemmer/go-junit-report v0.9.1 // indirect
-	github.com/karrick/godirwalk v1.12.0
+	github.com/jacobsa/fuse v0.0.0-20200323075136-ffe3eb03daf9
+	github.com/jmespath/go-jmespath v0.3.0 // indirect
+	github.com/karrick/godirwalk v1.15.5
 	github.com/kylelemons/godebug v1.1.0 // indirect
+	github.com/mailru/easyjson v0.7.1 // indirect
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
-	github.com/nightlyone/lockfile v0.0.0-20180618180623-0ad87eef1443
+	github.com/mitchellh/mapstructure v1.2.2 // indirect
+	github.com/nightlyone/lockfile v1.0.0
 	github.com/onsi/ginkgo v1.10.1 // indirect
 	github.com/onsi/gomega v1.7.0 // indirect
-	github.com/opentracing/opentracing-go v1.0.2
+	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pelletier/go-toml v1.6.0 // indirect
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
 	github.com/rhysd/go-github-selfupdate v1.2.1
-	github.com/rogpeppe/go-internal v1.5.2 // indirect
 	github.com/segmentio/ksuid v1.0.2
+	github.com/sirupsen/logrus v1.5.0 // indirect
 	github.com/spf13/afero v1.2.2
+	github.com/spf13/cast v1.3.1 // indirect
 	github.com/spf13/cobra v0.0.6
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/spf13/viper v1.5.0
-	github.com/stretchr/testify v1.4.0
-	go.opencensus.io v0.22.2
-	go.uber.org/goleak v0.10.1-0.20191111212139-7380c5a9fa84
-	go.uber.org/zap v1.13.0
-	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073 // indirect
-	golang.org/x/exp v0.0.0-20191129062945-2f5052295587 // indirect
-	golang.org/x/lint v0.0.0-20191125180803-fdd1cda4f05f // indirect
-	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6 // indirect
-	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	golang.org/x/sys v0.0.0-20200302150141-5c8b2ff67527
-	golang.org/x/tools v0.0.0-20200306191617-51e69f71924f // indirect
-	google.golang.org/api v0.15.0
-	google.golang.org/appengine v1.6.5 // indirect
-	google.golang.org/genproto v0.0.0-20191206224255-0243a4be9c8f // indirect
-	google.golang.org/grpc v1.25.1 // indirect
+	github.com/spf13/viper v1.6.2
+	github.com/stretchr/testify v1.5.1
+	go.mongodb.org/mongo-driver v1.3.1 // indirect
```

Comment:
Where did this come from?

Reply:
viper is a direct dependency
testify is indeed a test depency, it could go away
the mongo-driver, mmh I don't know which package could call that. Maybe this is pulled indirectly by the go-openapi stuff....

I am proceeding to a go mod tidy (again!) and it disappears now.. Weird

Reply:
removed now. Thanks